src/Symfony/Component/Serializer/NameConverter/MetadataAwareNameConverter.php

```diff
@@ -18,7 +18,7 @@
 /**
  * @author Fabien Bourigault <bourigaultfabien@gmail.com>
  */
-final class MetadataAwareNameConverter implements NameConverterInterface
+final class MetadataAwareNameConverter implements NameConverterInterface, AdvancedNameConverterInterface
```

Comment:
```suggestion
final class MetadataAwareNameConverter implements AdvancedNameConverterInterface
```

`AdvancedNameConverterInterface` extends `NameConverterInterface`